manifests/install.pp

```diff
@@ -177,17 +178,42 @@
       }
     }
     default: {
+      case $facts['os']['family'] {
+        'AIX': {
+          if "${::python::version}" =~ /^python3/ { #lint:ignore:only_variable_string
```

Comment:
please don't add leading `::`. That is only required for Puppet 3 compatibility.